Classes/CardIODataEntryViewController.m

```diff
@@ -929,12 +968,29 @@ - (void)postalCodeDidChange:(id)sender {
   [self validate];
 }
 
+- (void)cardholderNameDidChange:(id)sender {
+  self.cardInfo.cardholderName = self.cardholderNameTextField.text;
+
+  if([CardIOCardholderNameTextFieldDelegate isValidCardholderName:self.cardInfo.cardholderName]) {
+    [self advanceToNextEmptyFieldFrom:self.cardholderNameTextField];
+    self.cardholderNameTextField.textColor = [CardIOTableViewCell defaultDetailTextLabelColorForCellStyle:[CardIOTableViewCell defaultCellStyle]];
+  } else if(self.cardholderNameTextField.text.length >= 175) {
```

Comment:
They define the max length for Expiry as 7 in both of the same places. I figured following their style was better than being potentially confusing with more organized code.